plugins/processors/geo/README.md

```diff
@@ -0,0 +1,32 @@
+# Geo Processor Plugin
+
+Use the `geo` processor to add tag with S2 cell ID token of specified [`cell level`][cell levels].
+
+The tag is used in `experimental/geo` Flux package functions.
+
+### Configuration
+
+```toml
+[[processors.geo]]
+  ## The name of the lat and lon fields
+  lat_field = "lat"
+  lon_field = "lon"
+
+  ## New tag to create
+  tag_key = "_ci"
+
+  ## Cell level (see https://s2geometry.io/resources/s2cell_statistics.html)
+  cell_level = 11
+
+  ## Log mismatches
+  log_mismatches = false
```

Comment:
We would want to remove this option from the final version since we don't add logging options to the plugins.  Skip the actual logging, should be easily enough to figure out the issue.